codewars9.py

```diff
@@ -0,0 +1,39 @@
+# https://www.codewars.com/kata/54e6533c92449cc251001667/train/python
+# I misunderstood the assignment and wrote code which only prints the first occurence of a character in a string.
+# I will repair it later.
```

Comment:
Ok, good luck with repairing and let me know if you need some help. I will just give you some "clean code" tips here below.